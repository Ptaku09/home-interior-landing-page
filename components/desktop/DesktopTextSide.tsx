const DesktopTextSide = () => {
  return <div className="fixed top-0 left-0 z-[2] h-full w-1/3 bg-gradient-to-tr from-zinc-700 via-zinc-700 to-zinc-600" />;
};

export default DesktopTextSide;
