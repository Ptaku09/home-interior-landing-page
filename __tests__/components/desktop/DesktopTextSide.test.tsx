import '@testing-library/jest-dom';
import { render } from '../../test-utils';
import DesktopTextSide from '../../../components/desktop/DesktopTextSide';

describe('Desktop text side component', () => {
  it('should render', () => {
    const tree = render(<DesktopTextSide />);

    expect(tree).toMatchSnapshot();
  });
});
