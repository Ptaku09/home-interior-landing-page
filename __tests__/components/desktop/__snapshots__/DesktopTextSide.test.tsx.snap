// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Desktop text side component should render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="fixed top-0 left-0 z-[2] h-full lg:w-2/5 xl:w-1/3 bg-gradient-to-tr from-zinc-700 via-zinc-700 to-zinc-600"
      />
    </div>
  </body>,
  "container": <div>
    <div
      class="fixed top-0 left-0 z-[2] h-full lg:w-2/5 xl:w-1/3 bg-gradient-to-tr from-zinc-700 via-zinc-700 to-zinc-600"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
